pootle/settings/20-backends.conf

```diff
@@ -20,28 +20,47 @@ DATABASES = {
 
 # Cache Backend settings
 #
-# By default we use Django's local-memory cache which is only suitable
-# for small deployments.
+# By default we use Redis as our main cache backend as we potentially rely on
+# features specific to Redis.  Other backends will likely not work.
 #
-# If you are going to deploy a real server memcached is preferred. For more
-# information, check
+# For more # information, check
 # http://docs.djangoproject.com/en/dev/topics/cache/#setting-up-the-cache
+# and https://github.com/sebleier/django-redis-cache#usage
 CACHES = {
     'default': {
-        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
-        'LOCATION': 'pootlecache',
+        'BACKEND': 'redis_cache.RedisCache',
+        'LOCATION': 'localhost:6379',
         'TIMEOUT': 60,
         'OPTIONS': {
-            'MAX_ENTRIES': 65536,
-            'CULL_FREQUENCY': 16,
-        }
-    }
+            'DB': 1,
+            'PARSER_CLASS': 'redis.connection.HiredisParser',
```

Comment:
The redis-py parser should work fine, and unless there's a measured bottleneck here I wouldn't worry about doing premature optimizations at the cost of an extra C dependency that needs to be compiled.